conf/test.config

```diff
@@ -14,13 +14,17 @@ params {
     config_profile_name        = 'Test profile'
     config_profile_description = 'Minimal test dataset to check pipeline function'
 
+    max_cpus   = 2
+    max_memory = '6.GB'
+    max_time   = '6.h'
     // Limit resources so that this can run on GitHub Actions
 
     // Input data
     // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
     // TODO nf-core: Give any required params for the test so that command line flags are not needed
     input = "${projectDir}/assets/test.yaml"
     groups = 10
+    polishing_on = true
 }
 
 process {
```

Comment:
Everything under `process` needs to be moved to `conf/modules.config`. If it is in `conf/test.config` your tests will run correctly, but when you actually get to production, it will fail. Everything that you want implemented in production like the `longranger` settings should be moved to `modules.config` and removed from here as it would be redundant.